contentcuration/contentcuration/assets/shared/client.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+
+const client = axios.create({
+  xsrfCookieName: 'csrftoken',
+  xsrfHeaderName: 'X-CSRFToken',
+});
+
+client.interceptors.response.use(
+  response => response,
+  error => {
+    let message;
+    let url;
+    let config;
+    if (error.response) {
+      config = error.response.config;
```

Comment:
This was as direct a port as I could muster of the current error tracking that we have attached to jquery ajax.

Reply:
I don't think it should be necessary anywhere, so yes we should eliminate it completely.